src/python/pants/backend/shell/target_types.py

```diff
@@ -301,6 +301,17 @@ class ShellCommandToolsField(StringSequenceField):
     )
 
 
+class ShellCommandExtraEnvVarsField(StringSequenceField):
+    alias = "extra_env_vars"
+    help = softwrap(
+        """
+        Additional environment variables to include in the shell process.
+        Entries are strings in the form `ENV_VAR=value` to use explicitly; or just
+        `ENV_VAR` to copy the value of a variable in Pants's own environment.
+        """
+    )
```

Comment:
It's probably worth extracting this help string to a templating function, maybe in `src/python/pants/engine/environment.py` ? It's duplicated in at least `src/python/pants/backend/python/target_types.py` and `src/python/pants/core/goals/test.py`.

Reply:
Oh... or defining a `Field` superclass that provides it? Maybe overkill.

Reply:
Might be done as a follow up as well.